environments.json

```diff
@@ -1458,10 +1458,25 @@
   "node7": {
     "full": "Node.js",
     "family": "Node.js",
-    "short": "Node 7",
+    "short": "Node 7.0-7.5",
```

Comment:
I don't know the exact logic used to inherit stuff between different versions of the same thing. That is, Node versions are defined to equal specific Chrome versions and have only differences specified and as far as I see all Node versions are mentioned separately in such cases (just look at the two places where I added `node76`).

In any case, what would you propose? `7.0-7.5` catches the intent perfectly.